src/legendhpges/icpc.py

```diff
@@ -0,0 +1,235 @@
+"""Build the inverted coaxial point contact crystal template."""
+import json
+import logging
+from math import pi, tan
+
+import numpy as np
+
+# Third-party imports
+import pyg4ometry as pg4
+
+
+class icpc:
+    """Define ICPC Germanium template."""
+
+    def __init__(self, jsonfile, reg, materials):
+        """
+        Create ICPC detector logical volume (LV).
+
+        Parameters
+        ----------
+        jsonfile : str
```

Comment:
Once function arguments and return value are type annotated, type hints can be removed from docstrings (Sphinx will automatically add them to the HTML docs).